httpbp/errors.go

```diff
@@ -64,6 +65,11 @@ const (
 	DefaultErrorTemplateName = "httpbp/error"
 )
 
+// ClientMiddleware errors are returned if a known error is occuring during the middleware processing.
```

Comment:
grammar nit: remove "are"?

Reply:
No it's required by client spans: https://pkg.go.dev/github.com/reddit/baseplate.go/tracing#SpanTypeOption.Type